fares-data-build-tool/fdbt-site
# Feature/tfn 379 select journey selection

# Description

For the return single service we require an inbound and outbound journey selection which replaces the direction page for this part of the journey.

As part of this pr i have created a directionDropdown component which we can reuse.

https://infinityworks.atlassian.net/browse/TFN-379

# Testing instructions

- Start the journey 
- On the fareType page selected "Return - Single Service"
- Service page make a selection
- Should redirect to the new page which is the Select Journey Direction
- Testing for errors - such as selecting one dropdown and not another
- As there is no matching page for outbound and inbound yet the journey will not be completed

Lighthouse test
<img width="859" alt="Screenshot 2020-04-27 at 10 56 20" src="https://user-images.githubusercontent.com/1183416/80359465-c5d76400-8875-11ea-965c-60f58076fdec.png">



# Type of change


# Checklist:
